packages/evm/bindings/src/lib.rs

```diff
@@ -498,10 +498,10 @@ impl JsEvmWrapper {
         &mut self,
         node_env: Env,
         address: JsString,
-        slot: JsString,
```

Comment:
We need to pass a U256 value else we lose precision, so we can either keep `JsString` or alternatively pass a `JsBigInt`